CHANGES.md

```diff
@@ -9,6 +9,7 @@ Change Log
 * Refactored Csv, AbsItt and Sdmx-Json catalog items to depend on a common `TableCatalogItem`. Deprecated `CsvCatalogItem.setActiveTimeColumn` in favour of `tableStructure.setActiveTimeColumn`.
 * Error in geocoding addresses in csv files now shows in dialog box.
 * Fixed css styling of the timeline and added padding to the feature info panel.
+* Enhanced json support to recognise JSON5 format everywhere.
```

Comment:
This changelog entry isn't really true, is it?  Because `loadJson` does not support JSON5.  I've already merged this PR though so maybe you could tweak the wording in master?